Cloud Application Manager/DR Readiness/Create Node Pair.md

```diff
@@ -0,0 +1,51 @@
+{{{
+  "title": "Create Node Pair",
+  "date": "09-26-2019",
+  "author": "Sharon Wang",
+  "attachments": [],
+  "contentIsHTML": false
+}}}
+
+**In this article:**
+
+* [Overview](#overview)
+* [Audience](#audience)
+* [Prerequisites](#prerequisites)
+* [Create a Node Pair](#create-node-pair)
+
+### Overview <a name="overview"></a>
```

Comment:
Please, remove all the anchor links in titles, they will be automatically created by the Markdown parser